keras_nlp/models/mistral/mistral_layer_norm.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2023 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from keras_nlp.backend import keras
+from keras_nlp.backend import ops
+
+
+class MistralLayerNormalization(keras.layers.Layer):
```

Comment:
Can we deprecate this eventually for https://github.com/keras-team/keras/blob/21a3c0b7ce2bf738d81e3737aedba1a1ebd93c8b/keras/layers/normalization/layer_normalization.py#L85 ? If so, we still have to wait for dropping Keras 2, but we can leave a note.

Reply:
Do you want me to deprecate now? Or maybe add a TODO so we can get back to it once support for Keras 2 is dropped.

Reply:
Done.